spec/package_branch_spec.rb

```diff
@@ -34,12 +34,12 @@ def call_rake(*args)
       shared_package = FactoryGirl.create(:package, :package_branch_id => branch1.id, :shared => true)
       package = FactoryGirl.create(:package, :package_branch_id => branch2.id)
       shared_branches = PackageBranch.shared
-      
+
       shared_branches.count.should == 1
-      shared_branches.first.id.should == shared_package.id
+      shared_branches.first.id.should == shared_package.package_branch_id
```

Comment:
This is test I was talking about. It was comparing the package branch id to the package id...